src/components/WriteButton/WriteButton.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import WriteButton from "./WriteButton";
+
+describe("WriteButton", () => {
+  test("Should be a button", () => {
+    render(<WriteButton />);
+
+    expect(screen.getByRole("button")).toBeInTheDocument();
+  });
+
+  test("Should execute onClick", async () => {
+    const onClickStub = jest.fn()
+
+    render(<WriteButton onClick={onClickStub} />);
+
+    const button = await screen.findByRole("button");
+
+    fireEvent.click(button);
+    expect(onClickStub).toBeCalled()
```

Comment:
when testing that a function was called I like to test the it was called a exact number of time with the proper arguments.